src/main/java/com/targetindia/DigitizeTimeTable/model/Admin.java

```diff
@@ -0,0 +1,14 @@
+package com.targetindia.DigitizeTimeTable.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Admin {
+    int admin_id;
```

Comment:
use camelCase for names of fields, methods, and local variables. So it would be better to rename fields like admin_id, admin_name, admin_contact to adminId, adminName, adminContact, etc.